### ISSUE TITLE
Notification with tag works only once

### ISSUE THREAD

**Issue by manix** [electron/electron#11189]
After creating a notification and passing a tag, then subsequent notifications with the same tag don't show up for the entire lifespan of the app.

* Electron version: 1.8.1
* Operating system: Windows 7 x64

### Expected behavior

New notifications should open after the last one has closed.

### Actual behavior

Nothing happens

### How to reproduce

  $ git clone https://github.com/electron/electron-quick-start .
  $ npm install
  $ npm start

Then in the app open dev tools and paste this in the console:

    new Notification("test", { tag: "test" });

Wait for the notification to disappear and paste it again, observe nothingness.

**Comment by MarshallOfSound** [comment]
For clarity this is not a 1.8.x regression, it only occurs on Windows 7 and it's due to how we attempt to de-duplicate notification objects based on there tag in our windows 7 notification presenter

**Comment by gasi-signal** [comment]
We ran into this as well for the Signal Desktop app.

**Repro**


![windows-7-notifications-static-tag](https://user-images.githubusercontent.com/36971170/39647642-c3a7f7e2-4fad-11e8-803c-9ca1e7010585.gif)

**Code**

```javascript
let counter = 0;
let intervalId = setInterval(
  () => new Notification(counter++, { tag: "static-tag" }),
  1000
);
```

**Comment by sofianguy** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron v2.0.x or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons.

Thanks in advance! Your help is appreciated.

**Comment by sofianguy** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.

**Comment by r0stig** [comment]
Hello,

I just want to point out that I still get this bug on electron 4.1.4.

I'm able to reproduce the bug with the the code @gasi-signal posted and a clean electron-quick-start-repo.

**Comment by r0stig** [comment]
Just letting you know the bug is still present in 5.0.0.
